src/client/core/utils/array.js

```diff
@@ -0,0 +1,71 @@
+export function toArray (obj) {
+    return Array.prototype.slice.call(obj);
```

Comment:
Don't use `slice`, just copy items manually. Also don't use it for the `arguments` object. because of https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#32-leaking-arguments. Just copy items inline there and add appropriate comment with this doc